charts/kapacitor/templates/deployment.yaml

```
@@ -49,6 +49,12 @@ spec:
             secretKeyRef:
               key: influxdb-password
               name: {{ .Values.existingSecret }}
+        - name: KAPACITOR_SLACK_URL
+          valueFrom:
+            secretKeyRef:
+              key: kapacitor-slack-url
+              name: {{ .Values.existingSecret }}
+              optional: true
```

Just a thought - it the `kapacitor-` prefix of the key needed to distinguish it from another Slack URL? I mean influxdb-related keys do not have such prefix.

And second, given there can be multiple slack clients, then, similarly to influxdb env vars injection, having `_0_` index in the env var name would seem like using the same [correct naming style](https://docs.influxdata.com/kapacitor/v1.6/administration/configuration/#map-configuration-properties-to-environment-variables), I think, ie something like:

```suggestion
        - name: KAPACITOR_SLACK_0_URL
          valueFrom:
            secretKeyRef:
              key: slack-url
              name: {{ .Values.existingSecret }}
              optional: true
```
?

---

Hey, @alespour thanks for looking into this. Good point I'll remove the `kapacitor-` prefix to uniformize the secret keys.  

After reading the [naming style ](https://docs.influxdata.com/kapacitor/v1.6/administration/configuration/#map-configuration-properties-to-environment-variables) for the environment variables I thought `KAPACITOR_SLACK_URL` would match the path in the configuration file, but I missed the fact that we can have multiple Slack clients. Let me test that and commit a fix.